verification.py

```diff
@@ -71,6 +71,18 @@ def _plot_qq_compare(self, data1, data1_name, data2, data2_name):
         pyplot.savefig(f, dpi=72)
         pyplot.close("all")
 
+        sns.kdeplot(data1, color="b", fill=True, legend=False)
```

Comment:
Could these be shown on the same axes in different colours? The point is to compare the distributions, right?